test/local/senders/index.js

```diff
@@ -285,9 +287,44 @@ describe('lib/senders/index', () => {
           .then(() => {
             const args = email._ungatedMailer.unblockCodeEmail.getCall(0).args
             assert.equal(args[0].email, EMAILS[1].email, 'email correctly set')
-            assert.equal(args[0].ccEmails.length, 1, 'email correctly set')
-            assert.equal(args[0].ccEmails[0], EMAILS[1].email, 'cc email correctly set')
-            assert.equal(errorBounces.check.callCount, 3)
+            assert.equal(args[0].ccEmails.length, 0, 'email does not appear twice')
```

Comment:
Previously the email would appear in both "to" and "cc" fields, removing this duplication was a happy side-effect of the refactoring here.